src/js/components/Box/StyledBox.js

```diff
@@ -221,6 +236,7 @@ const StyledBox = styled.div`
   ${props => props.round && roundStyle}
   ${props => props.textAlign && textAlignStyle}
   ${props => props.wrap && wrapStyle}
+  ${props => props.responsive && responsiveStyle}
```

Comment:
I thought we were going to contemplate something other than `responsive` as a property?